iacrcc/iacrcc.cls

```diff
@@ -144,72 +144,112 @@
-}{}
-
-% XMP (eXtensible Metadata Platform) is a mechanism proposed by Adobe
-% for embedding document metadata within the document itself. It should
-% be loaded before hyperref. We require a version later than 2019/03/14,
-% and this will issue a warning if that is not satisfied.
-\RequirePackage{hyperxmp}[2019/03/14]
 \setcounter{tocdepth}{2}
 
-\RequirePackage{hyperref}
-\hypersetup{pdflang=en,
-            urlcolor=blue,
-            colorlinks=true,
-            citecolor=black!70!green,
-            linkcolor=black!70!red}
-% Disable latexdiff commands in PDF links
-\pdfstringdefDisableCommands{%
-  \def\DIFadd#1{#1}%
-  \def\DIFdel#1{}%
-}
 % doclicense added support for the macro \doclicenseLongTextForHyperref in version 3.1.0 (2022/04/27)
 \@ifpackagelater{doclicense}{2022/04/25}{\@doclicense@LongTextForHyperreftrue}{}
 \@ifpackageloaded{biblatex}{}{\bibliographystyle{alphaurl}}
+
 \AtEndPreamble{%
   \ifpdftex
     % TU encoding is only available with XeTeX and LuaTeX.
     % Defaulting to T1 encoding.
+    % fontenc: Standard package for selecting font encodings
     \RequirePackage[T1]{fontenc} 
   \fi
-  %% PDF title and author
+  \ifpdftex
+    % hyperref: Extensive support for hypertext in LaTeX
+    \RequirePackage{hyperref}
+    %
+    % XMP (eXtensible Metadata Platform) is a mechanism proposed by Adobe
+    % for embedding document metadata within the document itself.
+    % We require a version later than 2019/03/14,
+    % and this will issue a warning if that is not satisfied.
+    % Load this *after* hyperref as a work-around for recent issues, see
+    % https://tex.stackexchange.com/questions/694586/why-does-hyperxmp-5-11-no-longer-generate-dccreator-for-authors-in-xmp
+    % for more information (only when using pdflatex)
+    \RequirePackage{hyperxmp}[2019/03/14]
+  \else
+    \RequirePackage{hyperxmp}[2019/03/14]
+    \RequirePackage{hyperref}
+  \fi
+  %
+  % Pass the keeppdfinfo option to hyperxmp to ensure the
+  % Info dictionary is retained which contain the author names
+  \hypersetup{keeppdfinfo}
   \if@anonymous
     \hypersetup{pdfauthor={hidden for submission}}
   \else
-    \hypersetup{pdfauthor=\IACR@listofauthors}
+    \hypersetup{pdfauthor={\IACR@listofauthors}}
   \fi
-  \hypersetup{pdftitle={\@plaintitle}}
-  \if\@IACR@title@subtitle\empty
-  \else
+  \ifdefined\@IACR@title@subtitle@defined
     \hypersetup{pdfsubtitle={\@IACR@title@subtitle}}
   \fi
-  % other PDF metadata
+  \ifx\IACR@text@keywords\@empty\else
+    \hypersetup{pdfkeywords={\IACR@text@keywords}}
+  \fi
+  \hypersetup{pdftitle={\@plaintitle},%
+              pdflang=en,%
+              urlcolor=blue,%
+              colorlinks=true,%
+              citecolor=black!70!green,%
+              linkcolor=black!70!red%
+             }
   \ifcsstring{@IACRversion}{final}{%
     \hypersetup{%
       pdfpubstatus=VoR,
-      pdfdoi=\IACR@DOI,%
-      pdfissn=\IACR@ISSN,%
+      pdfdoi={\IACR@DOI},%
+      pdfissn={\IACR@ISSN},%
```

Comment:
This really should be `pdfeissn=...`. ISSNs can be both print and electronic - we don't plan to have a print ISSN as far as I know, though perhaps the `iacrcc.cls` should support both as `\IACR@EISSN` and `\IACR@PISSN`.